packages/ai/src/generate-object/output-strategy.ts

```diff
@@ -29,7 +29,7 @@ import {
 import { ObjectStreamPart } from './stream-object-result';
 
 export interface OutputStrategy<PARTIAL, RESULT, ELEMENT_STREAM> {
-  readonly type: 'object' | 'array' | 'enum' | 'no-schema';
+  readonly type: 'object' | 'array' | 'enum' | 'no-schema' | 'record';
```

Comment:
can this be part of 'object'? adding a new type for record is not a path i want to take